flink-formats/flink-avro/src/test/java/org/apache/flink/formats/avro/AvroRowDataDeSerializationSchemaTest.java

```diff
@@ -222,12 +234,14 @@ void testSerializeDeserializeBasedOnNestedSchema() throws Exception {
         AvroRowDataSerializationSchema serializationSchema =
                 new AvroRowDataSerializationSchema(
                         rowType,
-                        AvroSerializationSchema.forGeneric(nullableOuterSchema),
+                        AvroSerializationSchema.forGeneric(
```

Comment:
With the public BINARY-default methods maintained, you might want to revert these. 

This test doesn't work with JSON encoding?

Reply:
good point - updated the test in the same way with ParameterizedTest